### DISCUSSION TITLE
Smartmeter Firmware gelöscht

### DISCUSSION THREAD

**Discussion by firehawk1979** [arendst/Tasmota#22390] [Q&A]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by firehawk1979** [comment]
Ich habe beim Update leider mir die Firmware gecrasht, und nun habe ich das Problem das ich auf meinen ESP-C3 keine Tasmota Smartmeter Firmware flashen kann da ich leider auch keine .bin Datei finde. Vielleicht kann mir hier einer eine Datei zur verfügung stellen, oder vielleicht behilflich sein wie ich eine erstellen kann.

Vielen Dank schonmal im voraus

Marcel

**Comment by HofeBY** [comment]
https://github.com/HofeBY/Tasmota-Firmware

**Comment by firehawk1979** [comment]
Der Bitshake Support hat mir die Orginal-Firmware ota aufgespielt.

Danke an alle.
